app/src/main/java/io/kuenzler/whatsappwebtogo/WebviewActivity.java

```diff
@@ -237,7 +237,7 @@ public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request
                 final String url = request.getUrl().toString();
```

Comment:
URL url = request.getUrl();

Reply:
Hi @92lleo, 

Thanks for reviewing my commit and I think this is a good idea. I've changed the String object to the `android.net.Uri` type returned from `request.getUrl()`.